src/components/key-value-label/key-value-label.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import css from './key-value-label.css';
```

Comment:
Just being nitpicky here, most React examples call this `styles` as oppose to `css`. I don't care either way, just pointing it out.